lib/src/map/options/map_gestures.dart

```diff
@@ -265,9 +252,6 @@ abstract class InteractiveFlag {
   /// Enable panning with a single finger or cursor
   static const int drag = 1 << 0;
 
-  /// Enable fling animation after panning if velocity is great enough.
-  static const int flingAnimation = 1 << 1;
```

Comment:
May want to consider moving the latter bits down, but not important (especially if we are going to remove all this anyway).

Reply:
We can if you want, although would be another breaking change. Let me know if you want me to change them.